recipes/anyio/meta.yaml

```diff
@@ -1,41 +0,0 @@
-{% set name = "anyio" %}
```

Comment:
Please do not remove other recipes

Reply:
Even ones that were removed from rebasing on the master?